sql-plan-replayer.md

```diff
@@ -196,6 +196,11 @@ mysql> show stats_meta;
 
 After the scene is loaded and restored, you can diagnose and improve the execution plan for the cluster.
 
+> **Note:**
+>
+> If you use `mysql` and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```

Comment:
```suggestion
> If you use `mysql` command line utility, and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```

Reply:
This text was copied from the FAQ section of https://docs.pingcap.com/tidbcloud/tidb-cloud-import-local-files. Should I add "command line utility" there also? Thanks

Reply:
Why not :) does not hurt to be more clear. Grepping "command line" and "mysql" gave some examples like `MySQL command line` in: https://docs.pingcap.com/tidbcloud/sql-statement-kill#mysql-compatibility
But it is not strictly necessary, it is mostly a suggestion :)

Reply:
Totally agree! Thanks for the suggestion. And.....done.